src/lib/components/ManagePanel.js

```diff
@@ -55,9 +55,22 @@ export default class ManagePanel extends React.Component {
     return url;
   }
 
+  getDate(time) {
+    if (time === 'next') {
+      return 'Next time';
+    }
+    return moment(time).format('ddd, MMM D') || 'Later'
+  }
+
+  getTime(time) {
+    if (time === 'next') {
+      return 'Firefox opens';
+    }
+    return moment(time).format('[@] ha') || '';
+  }
```

Comment:
Similarly, if you've snoozed a tab to a date in the future and then edit the time, you seemingly can't edit the reminder for "Next time Firefox opens".